pkg/core/xds/inspect/attachments_test.go

```diff
@@ -301,12 +302,12 @@ var _ = Describe("GroupByAttachment", func() {
 				Retries: core_xds.RetryMap{
 					"backend": &core_mesh.RetryResource{Meta: meta1},
 				},
-				Dynamic: map[core_model.ResourceType]core_xds.TypedMatchingPolicies{
-					core_mesh.TrafficLogType: {
-						ServicePolicies: map[core_xds.ServiceName][]core_model.Resource{
-							"redis": {
-								&core_mesh.TrafficLogResource{Meta: meta6},
-							},
+			},
+			policyPlugins: core_xds.PluginOriginatedPolicies{
```

Comment:
Just a bit confusing that we are using `PluginOriginatedPolicies` with old policies

Reply:
Are we?

Reply:
We're using both right? `GroupByAttachment` takes both `PluginOriginatedPolicies` (new policies) and `MatchedPolicies` (old policies) and then in (e.g.) `getInboundMatchedPolicies` we combine them.